recipes/mbits-args/all/conanfile.py

```diff
@@ -1,103 +1,127 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import (
+    apply_conandata_patches,
+    export_conandata_patches,
+    get,
+    copy,
+    rm,
+    rmdir,
+)
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 import os
 
 
+required_conan_version = ">=1.53.0"
+
+
 class MBitsArgsConan(ConanFile):
     name = "mbits-args"
-    description = "Small open-source library for program argument parser, inspired by Python's `argparse`, " \
-                  "depending only on the standard library, with C++17 as minimum requirement."
-    homepage = "https://github.com/mbits-libs/args"
+    description = (
+        "Small open-source library for program argument parser, inspired by Python's `argparse`, "
+        "depending only on the standard library, with C++17 as minimum requirement."
+    )
     license = "MIT"
-    topics = ("command-line", "commandline", "commandline-interface",
-              "program-arguments", "argparse", "argparser", "argument-parsing")
-
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mbits-libs/args"
+    topics = (
+        "command-line",
+        "commandline",
+        "commandline-interface",
+        "program-arguments",
+        "argparse",
+        "argparser",
+        "argument-parsing",
+    )
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
-    generators = "cmake"
-    exports_sources = "CMakeLists.txt"
-
-    _cmake = None
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
 
-    _compilers_minimum_version = {
-        "gcc": "8",
-        "clang": "7.0",
-        "Visual Studio": "16",
-        "apple-clang": "10.0",
-    }
+    @property
+    def _min_cppstd(self):
+        return 17
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _compilers_minimum_version(self):
```

Comment:
I am not familiar with this project, could you show me why the versions were all bumped? https://github.com/conan-io/conan-center-index/pull/14866/files#diff-cc6427f36037d8d1c963d09b4c8208f12d3201cf4a254f79b2f751987e72a281L24-L29

Reply:
I took a "template" from libs created from new cmake templates (PRs #14785 and #14788). I'll downgrade the comps back and see what CI has to say (the code did not change, after all)

Reply:
✔️